app/src/main/java/de/westnordost/streetcomplete/quests/sport/AddSport.java

```diff
@@ -0,0 +1,48 @@
+package de.westnordost.streetcomplete.quests.sport;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddSport extends SimpleOverpassQuestType
+{
+	@Inject public AddSport(OverpassMapDataDao overpassServer)
+	{
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes, ways with leisure=pitch and !sport";
```

Comment:
> multipolygons

Ah ok, I see

> Why not check for deprecated tags, but this shouldn't be another quest ?

Because, how would that quest look like to the user? It doesn't make sense to inform the user that this pitch has been tagged with a deprecated tag and so he should resurvey it etc pp. To the user, the reasons why this quest is displayed are irrelevant - he should just answer the question.

So, it would look exactly the same as this one and also behave the same: . The idea is that the quest would just treat pitches that are tagged as team_handball or hockey as if it hasn't been tagged yet. This is not so important for team_handball, as this is really just a synonym but more so with hockey, because in the Americas, with "hockey" probably ice hockey is meant while elsewhere, probably field hockey is meant. These two can be distinguished at first glance ( ice / no ice).